src/Bicep.Core/FileSystem/InMemoryFileResolver.cs

```diff
@@ -95,7 +95,23 @@ public string GetRelativePath(string relativeTo, string path)
 
         public IEnumerable<Uri> GetDirectories(Uri fileUri, string pattern)
         {
-            return Enumerable.Empty<Uri>();
+            if (!fileUri.IsFile)
```

Comment:
@anthony-c-martin if I don't do ```return Enumerable.Empty()``` here the following test breaks https://github.com/Azure/bicep/blob/9a45fd99264df606f4fb36ef660ad1372b3f7729/src/Bicep.LangServer.IntegrationTests/CompletionTests.cs#L1187 

Any ideas on how to fix this?

Reply:
I think you just need to update the assertion:
```csharp
completions.Should().SatisfyRespectively(
    x => x.Label.Should().Be("mod with space.bicep"),
    x => x.Label.Should().Be("percentage%file.bicep"),
    x => x.Label.Should().Be("already escaped.bicep"),
    x => x.Label.Should().Be("folder with space"));
```